doc/CollisionSolverIsaac.doc.md

```diff
@@ -0,0 +1,49 @@
+# Collision Solver for Isaac
+
+## Label
+
+`collision`
+
+## Parameters
+
+| Label | Type | Size/Struct | Description |
+| ----- | ---- | ----------- | ----------- |
+| b | register | 1 byte | the 2 left bits tell whether the collision happens during (respectively) horizontal or vertical movement |

```

Comment:
Same things here, I'm afraid we can't have this information with the current collision function.